prime_factor/prime_factor.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <limits.h>
+
+int generate_primes(int num) {
+    int cnt = 0;
+    printf("Prime Factors: ");
```

Comment:
yes sir, so i have to pass num, n , prime_factor according to you in the function